pkg/kubelet/rkt/rkt.go

```diff
@@ -417,12 +417,12 @@ func setSupplementaryGIDs(app *appctypes.App, podCtx *api.PodSecurityContext) {
 
 // setApp merges the container spec with the image's manifest.
 func setApp(app *appctypes.App, c *api.Container, opts *kubecontainer.RunContainerOptions, ctx *api.SecurityContext, podCtx *api.PodSecurityContext) error {
-	// Override the exec.
-	if len(c.Command) > 0 {
-		app.Exec = c.Command
-	}
-	if len(c.Args) > 0 {
-		app.Exec = append(app.Exec, c.Args...)
+	// TODO(yifan): If ENTRYPOINT and CMD are both specified in the image,
+	// we cannot override just one of these at this point as they are already mixed.
```

Comment:
@yujuhong Not as far as I know. I think this might be cause some UX difference for running docker images with docker. We should doc it in the guide, but I don't know we would solve this. cc @jonboulle